deepchecks/base/display_suite.py

```diff
@@ -113,13 +135,13 @@ def display_suite_result(suite_name: str, results: List[Union[CheckResult, Check
         display_html(html, raw=True)
 
 
-def get_first_3(results: List[Union[CheckResult, CheckFailure]]):
-    first_3 = []
-    i = 0
-    while len(first_3) < 3 and i < len(results):
-        curr = results[i]
-        curr_name = curr.check.name()
-        if curr_name not in first_3:
-            first_3.append(curr_name)
-        i += 1
-    return ', '.join(first_3)
+# def get_first_3(results: List[Union[CheckResult, CheckFailure]]):
```

Comment:
Suite may have several similar checks, so this function is relevant for returning unique names., so the list of names will look funny)
If len of unique names is less than 3 than the other wording you proposed should be used

Reply:
> Suite may have several similar checks,

yup, I know, but set drops all duplicates

```
check_names = list(set(it.check.name() for it in results))
```